src/Uno.UI.RuntimeTests/MUX/Microsoft_UI_Xaml_Controls/ColorPicker/ColorPickerPage.xaml

```diff
@@ -0,0 +1,281 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<local:TestPage
```

Comment:
I believe most of the XAML here is copy/pasted from WinUI 3. Is it wise to change to tabs instead of spaces? When I originally ported the control I did NOT change the XAML at all so it could simply be copy/pasted to keep in sync without showing many spurious changes or require new indentation styles. Someday that sync could be automated even.

I've noticed for other WinUI 3 controls you seem to keep the WinUI 3 XAML indentation of two spaces as well.

Reply:
The problem is that VS will try to re-format when you paste code.

I faced this problem earlier when updating ColorPicker and at that time I added this `.editorconfig`: https://github.com/unoplatform/uno/blob/f3f7a51e2c09440e23d08247736067a4d2b6651c/src/Uno.UI/Microsoft/UI/Xaml/.editorconfig

I think we should add the same .editorconfig to `Uno.UI.RuntimeTests/MUX` directory

Reply:
Good call.